specification/components/r4/schemas/eRS-ServiceRequest.yaml

```diff
@@ -131,8 +53,12 @@ properties:
           example: '000000070000'
   status:
     type: string
+    description: 'Whilst a potential value to return is `draft`, we are not currently returning this value, but may chose to so in the future.'
```

Comment:
chose > choose.
to do so.
Table with descriptions for each of this status. This would be a proposed description.

Reply:
Check with BAs if this needs to be signed off.

Reply:
It doesn't need to be signed off - it's up to the developers how they want to word it, as long as the fundamental idea is there.